src/controller/datacontroller.js

```diff
@@ -169,9 +172,42 @@ export default class DataController {
 		return viewDocumentFragment;
 	}
 
+	/**
+	 * Sets initial input data parsed by the {@link #processor data processor} and
+	 * converted by the {@link #upcastDispatcher view-to-model converters}.
+	 * Initial data can be set only to document that {@link module:engine/model/document~Document#version} is equal 0.
+	 *
+	 * **Note** This method is {@link module:utils/observablemixin~ObservableMixin#decorate decorated} which is
+	 * used by e.g. collaborative editing plugin that syncs remote data on init.
+	 *
+	 * @fires init
+	 * @param {String} data Input data.
+	 * @param {String} [rootName='main'] Root name.
+	 */
+	init( data, rootName = 'main' ) {
+		if ( this.model.document.version ) {
+			/**
+			 * Cannot initialize already initialized data. Data should be initialized only once, during
```

Comment:
Data isn't the thing that you initialize. It's the editor. You initialize the editor (or model, in engine-oriented context) with data.

Reply:
Changed.